core/field_angle.js

```diff
@@ -334,8 +334,8 @@ Blockly.FieldAngle.prototype.updateGraph_ = function() {
 
 /**
  * Ensure that the input value is a valid angle.
- * @param {string|number=} newValue The input value.
- * @return {?number} A valid angle, or null if invalid.
```

Comment:
Why is this valid in field_number but not field_angle?

Reply:
Because field_angle inherits from field_text which defines this newValue as string.

Reply:
Turns out field_number didn't have the @extends jsdoc, but it is also a problem there. Thanks for pointing this out.